### DISCUSSION TITLE
QR error

### DISCUSSION THREAD

**Discussion by KTechDev4213** [gopro/labs#378] [General]
When I scan from the QR maker with a hero 7 black when It's set to linear the camera thinks it's wide

**Comment by dnewman-gpsw** [comment]
In testing on HERO7 here, all is working fine.  The command fL - Linear, correct sets to the lens to Linear.  Make sure the video mode selected supports Linear.

**Reply by KTechDev4213** [reply]
I did.

**Reply by dnewman-gpsw** [reply]
doesn't explain why it is working HERO7s here.   Scanning fL goes to Wide on your camera?  Factory reset, just in case.

**Reply by KTechDev4213** [reply]
I factory reset it, it still goes to wide.
What QR code generator did you use?

**Reply by dnewman-gpsw** [reply]
All qr generators are the same.  Use this webpage or QRControl.  The command **fL** goes to Linear here.

**Reply by KTechDev4213** [reply]
It still doesn't work.

**Reply by dnewman-gpsw** [reply]
![image](https://github.com/gopro/labs/assets/10856084/005486e0-61c5-423e-9f83-323c9f821f0e)

Try this.  If that doesn't work report your firmware version.

**Comment by KTechDev4213** [comment]
It worked!
________________________________
